assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -162,14 +162,13 @@ DropDown.propTypes = {
   options: PropTypes.array.isRequired,
   onChange: PropTypes.func.isRequired,
   disabled: PropTypes.bool,
-  ariaLabel: PropTypes.string,
+  ariaLabel: PropTypes.string.isRequired,
```

Comment:
This component is actually broken for the label. The label is passed as `ariaLabel` to `DropdownList`, but [it weirdly uses `aria-labelledby`](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/form/dropDown/list.js#L191) which takes an element id, not an actual label string, as thus the label is not applied.

A nice way to see what screen readers see, is to use the "Accessibility" tab in dev tools located here (showing a11y for font weight dropdown):

![Screen Shot 2020-05-26 at 9 22 32 PM](https://user-images.githubusercontent.com/637548/82965761-176b2f80-9f97-11ea-9bce-8e2ba020b6db.png)

You can see the browser assuming the button's name to be  "Regular" as it has no other information than the text content to base it on.